app/src/main/java/br/org/cesar/knot_setup_app/view/ThingActivity.java

```diff
@@ -0,0 +1,45 @@
+package br.org.cesar.knot_setup_app.view;
+
+import android.os.Bundle;
+
+import java.util.ArrayList;
+
+import br.org.cesar.knot_setup_app.view.scan.ScanFragment;
+import br.org.cesar.knot_setup_app.view.thingRegistered.ThingFragment;
+import br.org.cesar.knot_setup_app.model.Tab;
+import br.org.cesar.knot_setup_app.utils.Constants;
+
+public class ThingActivity extends BaseActivity {
+
+    private static final String HEADER_TITLE = "THING's";
+    private static final String LEFT_TAB_TITLE = "Unregistered";
+    private static final String RIGHT_TAB_TITLE = "Registered";
+
+    private ArrayList<Tab> tabs;
+
+    private ScanFragment scanFragment;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        tabs = new ArrayList<>();
+
+        scanFragment = new ScanFragment();
+        scanFragment.setService(Constants.OT_SETTINGS_SERVICE);
+
+        Tab leftTab, rightTab;
+        leftTab = new Tab(LEFT_TAB_TITLE, scanFragment);
+        rightTab = new Tab(RIGHT_TAB_TITLE, new ThingFragment());
+
+        tabs.add(leftTab);
+        tabs.add(rightTab);
+
+        setupHeader(HEADER_TITLE,tabs);
+    }
+
+    @Override
+    protected void onResume() {
```

Comment:
It is necessary to override this method?

Reply:
Same as above.